keps/sig-network/3070-reserved-service-ip-range/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Reserve Service IP Range For Dynamic and Static IP Allocation
+kep-number: 3070
+authors:
+  - "@aojea"
+owning-sig: sig-network
+participating-sigs:
+status: implementable
+creation-date: 2021-12-02
+reviewers:
+  - "@thockin"
+  - "@danwinship"
+approvers:
+  - "@thockin"
+  - "@danwinship"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.24"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.24"
+  beta: "v1.25"
+  stable: "v1.26"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: ServiceIPStaticSubrange
```

Comment:
@thockin do you have more "catchy"  name?

Reply:
You should know by now NOT to ask me to name things...

This name is fine with me,